pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [tool.ruff]
+target-version = "py311"
 line-length = 88
 
 select = ["E", "F", "I"]
 ignore = [ "E501" ]
 
 [tool.mypy]
-python_version = "3.10"
+python_version = "3.11"
```

Comment:
Please keep in mind that python 3.11 support in nixpkgs can still be spotty especially with some slower updating python packages.

Reply:
Thought this might be better than introducing an external dependency, since parsing toml is pretty much required for this

Reply:
I don't think this includes the python 3.11 interpreter itself. nix-update does not use third-party dependencies.